src/Components/Reports/Util.js

```diff
@@ -51,11 +51,13 @@ export const generateFilterText = (filters) => {
     const filterSeparatorOnLine = '\n';
     const hasStateFilter = filters?.stateFilter?.length > 0;
     const hasNameFilter =  filters?.hostnameOrId?.length > 0;
+    const hasOsFilter =  filters?.osFilter?.length > 0;
 
-    if (hasStateFilter || hasNameFilter) {
-        filterText = `\nFilters applied\n`;
+    if (hasStateFilter || hasNameFilter || hasOsFilter) {
+        filterText = `${filterSeparatorOnLine}Filters applied${filterSeparatorOnLine}`;
         filterText = hasStateFilter ? filterText.concat(`State: ${filters.stateFilter.toString()}${filterSeparatorOnLine}`) : filterText;
-        filterText = hasNameFilter ? filterText.concat(`Name: ${filters.hostnameOrId}`) : filterText;
+        filterText = hasNameFilter ? filterText.concat(`Name: ${filters.hostnameOrId}${filterSeparatorOnLine}`) : filterText;
+        filterText = hasOsFilter ? filterText.concat(`Operating System: ${filters.osFilter.toString()}${filterSeparatorOnLine}`) : filterText;
```

Comment:
We can show the selected `RHEL` versions in sorted order - more readable
```suggestion
        filterText = hasOsFilter ? filterText.concat(`Operating System: ${(filters.osFilter.sort()).toString()}${filterSeparatorOnLine}`) : filterText;
```

Reply:
@kgaikwad just a small suggestion here I feel we should move the `Name` above `State` to keep the order consistent with the systems Page UI. Let me know what you think!

Reply:
> We can show the selected RHEL versions in sorted order - more readable
Fixed!

> @kgaikwad just a small suggestion here I feel we should move the Name above State to keep the order consistent with the systems Page UI. Let me know what you think!

Moved Name above State filter while generating filter text.  Fixed existing test and added one with osFilter.